src/compiler/visitor.ts

```diff
@@ -461,7 +461,7 @@ namespace ts {
         const edgeTraversalPath = nodeEdgeTraversalMap[node.kind];
         if (edgeTraversalPath) {
             for (const edge of edgeTraversalPath) {
-                const value = (<Map<any>>node)[edge.name];
+                const value = edge && (<Map<any>>node)[edge.name];

```

Comment:
PR to remove commas manually in https://github.com/Microsoft/TypeScript/pull/8065, issue to add an ES3 transformation and remove them by the compiler in https://github.com/Microsoft/TypeScript/issues/7938